demo-projects/custom-fields/fields/MultiCheck/views/Controller.js

```diff
@@ -0,0 +1,24 @@
+const FieldController = require('@keystonejs/fields/Controller').default;
```

Comment:
Why are you using CommonJS here?

Reply:
Because it borked out when I didn't 😆

Reply:
That's very strange. The Controller should be able to be ESM